lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -45,6 +46,15 @@ def get_path(digest)
       full_path
     end
 
+    def spool_errors
+      path = "#{@path_to_dir}/#{@namespace}"
+      { :errors => File.exists?(path) && list_dirs(path).any? }
```

Comment:
I wonder if it would make sense to return number of corrupted reports so people can see if it's growing since last time they checked.

Reply:
on second though, it might be a separate key, so it could look like

{ :errors => true; :errors_count => 5 }
{ :errors => false; :errors_count => 0 }